imports/plugins/core/core/server/no-meteor/resolvers/index.js

```diff
@@ -17,9 +18,7 @@ export default {
   Date: GraphQLDate,
   DateTime: GraphQLDateTime,
   Money,
-  Mutation: {
-    echo: (_, { str }) => `${str}`
```

Comment:
You should add the `echo` mutation in `Mutation/index.js` so we don't lose it. It's useful for testing and training.

Reply:
Ahh, ok, I thought it was just there as kind of a placeholder since there was no `Mutation` in this package.